var/spack/repos/builtin/packages/ncl/package.py

```diff
@@ -36,9 +36,9 @@ class Ncl(Package):
 
     homepage = "https://www.ncl.ucar.edu"
 
-    version('6.3.0', '4834df63d3b56778441246303ab921c4',
+    version('6.4.0', 'a981848ddcaf1c263279648265f24766',
             url='https://www.earthsystemgrid.org/download/fileDownload.html?'
-                'logicalFileId=bec58cb3-cd9b-11e4-bb80-00c0f03d5b7c',
+                'logicalFileId=86b9bec2-fa01-11e6-a976-00c0f03d5b7c',
```

Comment:
Can you put the entire URL on a single line? Lines containing `https:` are exempt from line-length checks so it won't be a problem.